server/shared/types/terms/geneVariant.ts

```diff
@@ -40,9 +39,14 @@ export type GeneVariantGeneTerm = BaseTerm & {
 	stop?: number
 }
 
-export type GeneVariantTerm = GeneVariantCoordTerm | GeneVariantGeneTerm
+export type GeneVariantTerm =
+	| GeneVariantCoordTerm
+	| (GeneVariantGeneTerm & {
+			groupsetting: TermGroupSetting
+	  })
```

Comment:
Does this only add `groupsetting` property onto `GeneVariantGeneTerm`? If so, it should also be added onto `GeneVariantCoordTerm`.

Reply:
Yes, you are correct. I will be changing this to a super type in another PR.